easybuild/easyconfigs/p/PRINSEQ/PRINSEQ-0.20.4-foss-2015b-Perl-5.22.0.eb

```diff
@@ -0,0 +1,118 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+# Modified by Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'Tarball'
+
+name = 'PRINSEQ'
+version = '0.20.4'
+
+homepage = 'http://prinseq.sourceforge.net'
+description = """A bioinformatics tool to PRe-process and show INformation of SEQuence data."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = ['http://sourceforge.net/projects/prinseq/files/standalone/']
+sources = ['%(namelower)s-lite-%(version)s.tar.gz']
+
+perl = 'Perl'
+perlver = '5.22.0'
+versionsuffix = '-%s-%s' % (perl, perlver)
+
+dependencies = [
+    (perl, perlver, '-bare'),
```

Comment:
in the existing `PRINSEQ` easyconfig, the Perl version included is non-bare...

maybe go with 5.20.3 (since the required easyconfig is already there)?